src/modules/assessments/util.tsx

```diff
@@ -65,18 +65,6 @@ export const generateAssessmentPath = (
   });
 };
 
-export const assessmentRowLinkTo = (
```

Comment:
I removed this in favor of keeping the assessment link consistent. I think this means that now, any table column that links to an assessment will open it in a household context if applicable. I'm not 100% confident in this change, maybe there was a good reason for the Client assessments page to link to the individual assessment view? I'm torn about whether it seems better to be consistent or maintain the previous behavior, let me know what you think.

Reply:
This was a design decision so lets review with design before reverting it